src/events/messageCreate.ts

```diff
@@ -78,7 +82,7 @@ export const messageCreate = async (bot: ExtendedClient, message: Message) => {
     ) {
       if (
         (bot.user && message.mentions.has(bot.user)) ||
-        /melody/i.test(content)
+        (/melody/i.test(content) && message.type !== MessageType.Reply)
```

Comment:
```suggestion
        /melody/i.test(content) && message.type !== MessageType.Reply
```

Reply:
I have a question. Isn't it clearer with braces here?

Reply:
No, I messed up the suggestion. Should be:

```suggestion
        /melody/i.test(content)
```